modules/sources/transformers/pool-transformer.ts

```diff
@@ -18,14 +18,18 @@ export const poolTransformer = (poolData: JoinedSubgraphPool, chain: Chain) => {
                 amp: '10', // TODO just a place holder
             } as StableData;
             break;
+        // @ts-ignore – that's a not coming from the subgraph
```

Comment:
the `JoinedSubgraphPool` anyway needs to be generalized more once we move the v2 sync here

Reply:
true, fixing on the type itself